package.json

```diff
@@ -28,20 +28,22 @@
   },
   "homepage": "https://github.com/medialab/artoo",
   "devDependencies": {
-    "gulp": "^3.9.0",
+    "gulp": "^4.0.2",
     "gulp-artoo": "~0.1.0",
     "gulp-concat": "^2.5.2",
     "gulp-header": "^1.7.1",
     "gulp-jshint": "^1.11.2",
     "gulp-mocha": "^2.0.0",
-    "gulp-mocha-phantomjs": "^0.10.1",
+    "gulp-mocha-phantomjs": "^0.12.1",
     "gulp-rename": "~1.2.0",
     "gulp-replace": "~0.5.0",
     "gulp-uglify": "^1.1.0",
     "gulp-webserver": "^0.9.0",
     "run-sequence": "^1.0.2"
```

Comment:
Shouldn't you drop this dependency since you don't require it anymore?

Reply:
yes!